payment_cern/indico_payment_cern/tests/util_test.py

```diff
@@ -17,6 +17,10 @@
     (123, 456, 'x', 'Foo Bar',                         'XBARFOOc123r456'),
     (123, 456, '',  'FooVeryLongName BarVeryLongName', 'BARVERYLONGNAMEFOOVERYc123r456'),
     (123, 456, 'x', 'FooVeryLongName BarVeryLongName', 'XBARVERYLONGNAMEFOOVERc123r456'),
+    (123, 456, 'x', 'Björn Groß', 'XGROSSBJOERNc123r456'),
+    (123, 456, 'x', 'Søren Åkesson', 'XAAKESSONSORENc123r456'),
+    (123, 456, 'x', 'Tomáš Dvořák', 'XDVORAKTOMASc123r456'),
+    (123, 456, 'x', 'けんじ 中村', 'Xc123r456'),  # XXX: This results in an empty order ID
```

Comment:
Should we do something about this?

Reply:
You still have the event/registration id ref so it's fine. Not much we can do beyond that anyway...